plugins/GSdx/GSUtil.cpp

```diff
@@ -396,6 +396,34 @@ D3D_FEATURE_LEVEL GSUtil::CheckDirect3D11Level(IDXGIAdapter *adapter, D3D_DRIVER
 	return SUCCEEDED(hr) ? level : (D3D_FEATURE_LEVEL)0;
 }
 
+GSRendererType GSUtil::GetBestRenderer()
+{
+	CComPtr<IDXGIAdapter1> adapter;
+	DXGI_ADAPTER_DESC1 desc;
+	CComPtr<IDXGIFactory1> dxgi_factory;
+
+	if (!GSUtil::CheckDXGI() || !GSUtil::CheckD3D11())
+		return GSRendererType::DX9_HW;
```

Comment:
1 and 2 should be done.

about 61: I'll stick to Microsoft's recommendation. :P